### PR TITLE
Update docsum according to docker compose changes

### LINKED ISSUE #579: [ci-auto] adapt the docsum helmchart for new pipeline component
## GenAIExample DocSum compose.yaml got changed

Below files are changed in [this commit](https://github.com/opea-project/GenAIExamples/commit/eb91d1f054a8d8b65ac6fda0e039522bd877f7ac)

- DocSum/docker_compose/intel/cpu/xeon/compose.yaml
- DocSum/docker_compose/intel/hpu/gaudi/compose.yaml

Please verify if the helm charts and manifests need to be changed accordingly.

> This issue was created automatically by CI.

### PR DESCRIPTION
**Pull request by yongfengdu**
Added multimedia2text and video2audio service

## Description

The summary of the proposed changes as long as the relevant motivation and context.

## Issues

Fixes #579 

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by lianhao — APPROVED** [review]
Please fix the yaml lint issue, others seems fine to me

File: helm-charts/docsum/values.yaml [hunk 1]
```diff
@@ -10,9 +10,27 @@ LLM_SERVICE_HOST_IP: ""
 
 image:
   repository: opea/docsum
-  pullPolicy: IfNotPresent
+    #pullPolicy: IfNotPresent
```
Comment (ashahba) [inline helm-charts/docsum/values.yaml:13]: Please leave one space before `#` so it reads:
```
# pullPolicy: IfNotPresent
```

**Review by ashahba — COMMENTED** [review]
Looks with the exception of one minor linter issue

**Review by ashahba — APPROVED** [review]
LGTM!

**Pull request merged**